src/csharp/Microsoft.Spark.UnitTest/AssemblyLoaderTests.cs

```diff
@@ -45,5 +57,17 @@ public void TestAssemblySearchPathResolver()
                 AssemblySearchPathResolver.AssemblySearchPathsEnvVarName,
                 null);
         }
+
+        [Fact]
+        public void TestResolveAssemblyWithRelativePath()
+        {
+            _mockJvm.Setup(m => m.CallStaticJavaMethod(
+                "org.apache.spark.SparkFiles",
+                "getRootDirectory"))
+                .Returns(".");
+
+            AssemblyLoader.LoadFromFile = AssemblyLoadContext.Default.LoadFromAssemblyPath;
+            AssemblyLoader.ResolveAssembly(Assembly.GetExecutingAssembly().FullName);
```

Comment:
So, does this throw without the fix? Also, we can do `Assert.NoThrows` to be explicit?

Reply:
Yup.  Without the fix this will throw the exception that we saw.

Reply:
I didn't find a `Assert.NoThrows` method.  Also `Assert.DoesNotThrow` was removed from Xunit https://github.com/xunit/xunit/issues/188

Reply:
then can we at least verify the returned `Assembly`?